src/Sitegen/Program.cs

```diff
@@ -107,22 +108,33 @@ private Program(TopLevelConfig topLevelConfig)
 
         public static TopLevelConfig ReadConfig(string path)
         {
+            if (!File.Exists(path))
+            {
+                throw new ConfigurationException($"{path} does not exist");
+            }
+
```

Comment:
(One nice side feature of unit testing is that you tend to more carefully test various error scenarios when writing tests. This tends to improve the overall error handling of your software.)